code/style.css

```diff
@@ -1,12 +1,19 @@
 * {
   box-sizing: border-box;
+  font-family: "Montserrat", sans-serif;
+  color: #302f2f;
 }
 
 body {
+  height: 100%;
   margin: 0;
   padding: 0;
-  font-family: 'Montserrat', sans-serif;
-  background: #0026ff;
+  font-family: "Montserrat", sans-serif;
+  background-image: url("background-picture.jpg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
 }
 
 h1 {
```

Comment:
It is a little hard to read the heading on the page in contrast to the coffee beans. Maybe a background with some opacity could be used to solve this?